src/Models/LoginSession.php

```diff
@@ -221,6 +257,17 @@ public static function generate(Member $member, bool $persistent, HTTPRequest $r
         return $session;
     }
 
+    /**
+     * @param HTTPRequest|null $request
+     * @return void
+     */
+    public function updateLastAccessed(HTTPRequest $request = null)
+    {
+        $this->LastAccessed = DBDatetime::now()->Rfc2822();
+        $this->IPAddress = LoginSession::getAnonymizedIPFromRequest($request);
```

Comment:
We can't just suddenly change how we're storing the IP address in a minor release - there may be projects that rely on that information being available in its current form. At best this could be behind a configuration option to opt-in to anonymising the IP address.

Reply:
yes, agree, i'm happy to split the PR in two (one for the last accessed field, and one for the ip anonymization)

Reply:
Yes, please do split this out into two separate PRs. As is, I can see the IP anonymising change being merged in provided it's behind a configuration option, we offload the anonymisation itself to some third-party, and a matching PR is made to call that out in the changelog (and ideally the docs if there's an appropriate spot there somewhere, probably in the security section).

Reply:
ok will do